oci/caps/utils.go

```diff
@@ -79,7 +79,8 @@ func NormalizeLegacyCapabilities(caps []string) ([]string, error) {
 }
 
 // TweakCapabilities tweaks capabilities by adding, dropping, or overriding
-// capabilities in the basics capabilities list.
+// capabilities in the basics capabilities list. All capabilities are added
+// if privileged is true.
 func TweakCapabilities(basics, adds, drops []string, privileged bool) ([]string, error) {
```

Comment:
@austinvazquez We can probably reimplement this using `WithXXX` functions from containerd; there's some small bits that I think are missing;

- I don't think containerd exposes the `DefaultCapabilities`
- Containerd's code doesn't use `ALL` (for "all capabilities"), which we use to allow the user to specify "all" capabilities, except for XYZ.

I need to give it another look though, but would be nice if we could consume more of these options from containerd instead of replicating the same things.